sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -27,10 +27,11 @@ dotnet add package Azure.AI.TextAnalytics
 
 This table shows the relationship between SDK versions and supported API versions of the service:
 
-> Note that `5.2.0-beta.3` is the first version of the client library that targets the Azure Cognitive Service for Language APIs which includes the existing text analysis and natural language processing features found in the Text Analytics client library. In addition, the service API has changed from semantic to date-based versioning. This version of the client library defaults to the latest supported API version, which currently is `2022-04-01-preview`.
+> Note that `5.2.0` is the first stable version of the client library that targets the Azure Cognitive Service for Language APIs, which include the existing text analysis and natural language processing features found in the Text Analytics client library. In addition, the service API has changed from semantic to date-based versioning.
```

Comment:
```suggestion
> Note that `5.2.0` is the first stable version of the client library that targets the Azure Cognitive Service for Language APIs which includes the existing text analysis and natural language processing features found in the Text Analytics client library. In addition, the service API has changed from semantic to date-based versioning.
```

The comma is extraneous, and the subject of this sentence is singular so "includes" is correct.